content/json/menus/help.json

```diff
@@ -8,6 +8,23 @@
       "link": "/project-partners",
       "selectedPatterns": [
         "^/project-partners"
+      ],
+      "items": [
+        {
+          "id": "snite",
+          "label": "Snite Museum of Art",
+          "link": "/project-partners#snite"
```

Comment:
Unfortunately, I the "#" is not playing nicely with our links and doesn't resolve to this anchor in the page. Should we just make three separate pages for each partner OR we could just put some jump links on the top of the page and remove the links from the left side menu. Any suggestions here?